admin/app/view_models/workarea/admin/variant_view_model.rb

```diff
@@ -12,7 +12,7 @@ def inventory
     end
 
     def available_inventory
-      inventory.available
+      inventory.available_to_sell
```

Comment:
This seems like changing the meaning of this method, what was the idea?

Reply:
i honestly don't know for sure. 3 months is a long time. Looking at where it gets used I assume its a display thing in the admin, but would cause problems so I'll take it out and see if there is another change that can be made.